packages/terra-alert/docs/MoreInformation.md

```diff
@@ -20,7 +20,7 @@ This image shows the layout of the elements of the Alert component.
 |action |element  |optional  |null  |An action element to be added to the action section of the alert.|
 |children  |node or string  |optional  |''  |Child Nodes providing the message content for the alert. Can contain text and HTML.|
 |customIcon  |element  |optional  |null  |The icon to be used for an alert of type custom. This will not be used for any other alert types.|
-|customStatusColor |string  |optional  |''  |The status bar color to be used for an alert of type custom.|
+|customColorClass |string  |optional  |''  |The status bar color to be used for an alert of type custom.|
```

Comment:
@neilpfeiffer 

From a UX standpoint, should there be a default color if this is not supplied?